### PR TITLE
Add a document to config footer text

### PR DESCRIPTION
**Pull request by utchoang**
Related https://github.com/apache/cloudstack/pull/5628.
The documentation allows configuring a footer text at the Login screen.

### CODE REVIEW

**Comment by PaulAngus** [comment]
Inline with comment in https://github.com/apache/cloudstack/pull/5628 - It would seem to me more logical for the loginFooter text to be configured next to the Footer text.
And therefore the loginFooter text explanation would move up to the same documentation area as the Footer text.

Way down in the advanced plugins section seems a very odd place for it to end up.

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

File: source/adminguide/ui.rst [hunk 1]
```diff
@@ -185,18 +185,19 @@ Users can customize the CloudStack's user interface by means of a configuration
...
-========== ==================================================
-apiBase    Changes the suffix for the API endpoint
-docBase    Changes the base URL for the documentation
-appTitle   Changes the title of the portal
-footer     Changes the footer text
-logo       Changes the logo top-left side image
-banner     Changes the login banner image
-error.404  Changes the image of error Page not found
-error.403  Changes the image of error Forbidden
-error.500  Changes the image of error Internal Server Error.
```
Comment (sureshanaparti) [inline source/adminguide/ui.rst:199]: @utchoang couldn't see this property details here, please check.
http://qa.cloudstack.cloud/docs/WIP-PROOFING/pr/251/adminguide/ui.html#basic-ui-customization

Comment (utchoang) [reply source/adminguide/ui.rst:199]: @sureshanaparti Do these changes appear to be after you changed the base from master to 4.16?

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

**Comment by utchoang** [comment]
@blueorangutan docbuild

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

**Comment by sureshanaparti** [comment]
Doc looks good.

Doc-FooterTextConfig

**Review by sureshanaparti — APPROVED** [review]

**Pull request merged**